src/data_utils.jl

```diff
@@ -332,16 +332,23 @@ UnivariateFinite(::LightInterface, a...; kwargs...) =
 
 const UNIVARIATE_FINITE_VECTOR_DOCSTRING =
 """
-    UnivariateFiniteVector(scores, classes)
+    UnivariateFiniteVector(classes, p; pool=nothing, ordered=false)
 
 Container for UnivariateFinite elements optimised for efficiency.
 Accessing a single element will construct and return the corresponding
 UnivariateFinite lazily.
+
+Here the probabalities `p` should be an array with `size(p, 2) = N`,
```

Comment:
probabilities; and I would  suggest: `size(p, 2) = C` where `C = length(classes)` and rows sum to  one.

Reply:
I get your philosophical point. However these are probabilities in the strictest, mathematical sense (they sum to one and hence define a probability measure on the space of classes).  If we say "scores" we the also need to constantly say "and they must sum to one." Also, this is the terminology used throughout Distributions.jl, which we are extending. 

Point taken about the notation. It's a bit of a mess in this particular part of the code. Happy to go with your choice here.